src/components/Location/LocationSheet.tsx

```diff
@@ -252,6 +279,37 @@ export function LocationSheet({
           >
             {updateAssociation.isPending ? t("saving") : t("save")}
           </Button>
+          {location.status != "active" && (
```

Comment:
hmm 🤔  need to check whether this is the behavior we want. Meanwhile, reset the changes made for translation files.

Reply:
@YourBroCode Don't resolve, for other team members to see 👍